runtime/cudaq/matrix_op.h

```diff
@@ -176,12 +228,42 @@ class matrix_handler : public operator_handler {
 
   // predefined operators
 
+  /// @brief Constructs a product operator representing a number operator for
+  /// the given degree.
+  /// @param degree : The degree or power to which the number operator is
+  /// constructed.
+  /// @return A matrix_handler instance that encapsulates the
+  /// constructed number operator matrix.
   static matrix_handler number(std::size_t degree);
+  /// @brief Creates a parity operator using a product operator with
+  /// matrix_handler.
+  /// @param degree : The degree of the parity transformation, which may
+  /// correspond to the number of qubits or the order of the operation.
+  /// @return A product operator that encapsulates the parity transformation.
   static matrix_handler parity(std::size_t degree);
+  /// @brief Constructs a product operator representing the position operator.
+  /// @param degree : Specifies the operator's degree, which may determine the
+  /// approximation order or related numerical properties.
```

Comment:
I am getting really curious where you read up on operators. :) 
The degree here has nothing to do with any approximation order. It is merely the degree of freedom that the operators acts on. The description of param degree can be updated to be the same for all operators defined in handlers.